tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist = csslint,eslint,py27,pycodestyle,pylint
 
-[testenv]
+[testenv:py27]
```

Comment:
We only actually need these dependencies specified for the Python tests
(the other targets either don't need the deps or they install them on their own).

So we can speed up the `eslint` and `csslint` targets by omitting the installation there.